app/views/layouts/_messages.html.erb

```diff
@@ -0,0 +1,6 @@
+<% flash.each do | name, msg |%>
+  <div class="alert alert-<%= select_class_alert(name)%>">
```

Comment:
Consider being consistent, as here you're iterating with `name` and in the method definition you operate with `type`